### DISCUSSION TITLE
do not see committed file ..

### DISCUSSION THREAD

**Discussion by leonard-solomyak** [community/community#43863] [New to GitHub]
Good day, 

I lost ground .. recommitted the existing file to correct the description, but it does not appear in the history for this file ..
Instead my commits appear on top against my user avatar and seem without any file in it ..

![1 commiting existing file](https://user-images.githubusercontent.com/115936340/211593251-34135cb3-64ba-4749-bfbb-84e5c7271421.png)

![2 new commit is on top with my user avatar](https://user-images.githubusercontent.com/115936340/211593305-d817dba9-2ce4-47ad-97aa-6b874b4a9a24.png)

![3 new commit does not have file reference](https://user-images.githubusercontent.com/115936340/211593323-fe88177b-e9ec-4537-a359-7165e53ed88e.png)

![4 do not see the new commit in the history](https://user-images.githubusercontent.com/115936340/211593339-63dbf084-b1f6-400d-b157-10e66a8a787b.png)

**Comment by leonard-solomyak** [comment]
perhaps, it is because it was the file committed previously (just before) ?

**Comment by Vaishali14175** [comment]
I have avtaar...
